server.rb

```diff
@@ -1,22 +1,18 @@
 require "sinatra"
 require "sinatra/activerecord"
 
-
 set :views, Proc.new { File.join(root, "app/views") }
 
 get "/" do
   erb :sign_in
 end
 
-post "/sign_in" do
-
+post "/session/new" do
```

Comment:
Like with `/user/new`, we can treat signing in as a "resource" - a "session" that we create on login and destroy on logout - even though we won't persist this resource in the database.